nixos/lib/testing/interactive.nix

```
@@ -6,11 +6,31 @@ in
   options = {
     interactive = mkOption {
       description = mdDoc ''
-        Tests [can be run interactively](#sec-running-nixos-tests-interactively).
+        Tests [can be run interactively](#sec-running-nixos-tests-interactively)
+        using the program in the test derivation's `.driverInteractive` attribute.
```

As above, this seems to unnecessarily expose internals.

```suggestion
        Tests [can be run interactively](#sec-running-nixos-tests-interactively).
```

---

`.driverInteractive` is crucial for using the `interactive` submodule, so it is worth repeating here.

`sec-running-nixos-tests-interactively` documents the user interface. `.driverInteractive` isn't just part of the UI but part of `interactive`'s _relation to_ the UI, so it should be mentioned here alongside the link.

---

Ah now I found it in the manual section that was not part of the diff. Sorry for the noise.